### PR TITLE
Allow using Python scripts without bazel.

### PR DESCRIPTION
**Pull request by advaitjain**
Manually verified and documented the steps that I can follow to run Python code without Bazel.

BUG=http://b/204109200
NO_CHECK_TFLITE_FILES=manually ran the sync script.

### CODE REVIEW

File: docs/python.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!--ts-->
+   * [Using Bazel](#using-bazel)
+   * [Manual Setup Illustration](#manual-setup-illustration)
+
+<!-- Added by: advaitjain, at: Fri Oct 29 11:57:14 AM PDT 2021 -->
+
+<!--te-->
+
+Writing and using Python scripts from the TFLM repository is currently in the
+prototyping stage. As such, the instrucitons below are somewhat sparse and
```
Comment (njeffrie) [inline docs/python.md:10]: s/instructions (sp)

**Review by njeffrie — DISMISSED** [review]
Very cool! I'm not 100% clear on exactly what the steps in the shell script in python.md do, perhaps a quick chat could help me understand (mostly for my own benefit). Looks good overall with the one tiny spelling fix.

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,5 @@
 .vscode/
 *audio_frontend*
 *google*
+*__pycache__*
```
Comment (deqiangc) [inline .gitignore:6]: should pyc file be included as well?

File: docs/python.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+subject to change.
+
+
+* [TensorFlow Python style guide](https://www.tensorflow.org/community/contribute/code_style#python_style)
+
+
+# Using Bazel
+
+We use Bazel as our defaul build system for both Python and C++ and the
+continuous integration system uses bazel to run the Python unit tests.
```
Comment (deqiangc) [inline docs/python.md:20]: nit: just "python"?  I feel mentioning python and c++ together in this phrase together may lead people to infer c++ also use bazel as default, but I think for C++, make is at least as important as bazel. WDYT?

**Review by deqiangc — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (advaitjain) [reply .gitignore:6]: The pyc files are all within the __pycache__ directory.

File: docs/python.md (hunk 1 above)
Comment (advaitjain) [reply docs/python.md:10]: fixed

File: docs/python.md (hunk 3 above)
Comment (advaitjain) [reply docs/python.md:20]: good point. changed to mention Python only.

**Review by deqiangc — APPROVED** [review]

**Pull request merged**